tests/test_routes.py

```diff
@@ -107,6 +107,7 @@ def test_route_landing_export_link_visibility(
                     export_href=(
                         "/assess/assessor_export/TF/tr/ASSESSOR_EXPORT"
                     ),
+                    feedback_export_href="/assess/feedback_export/TF/tr",
```

Comment:
Need some unit tests that check whether the link is displayed or not, and also check error handling in `routes.py` for if application store is unavailable.